installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -3383,47 +3383,60 @@ update  cf_PolicyTable set policyDesc = 'Using a single IAM access key for IAM u
 
 INSERT  IGNORE INTO cf_PolicyTable (policyId, policyUUID, policyName, policyDisplayName, policyDesc, resolution, resolutionUrl, targetType, assetGroup, alexaKeyword, policyParams, policyFrequency, policyExecutable, policyRestUrl, policyType, policyArn, severity, category, autoFixAvailable, autoFixEnabled, allowList, waitingTime, maxEmailNotification, templateName, templateColumns, fixType, warningMailSubject, fixMailSubject, warningMessage, fixMessage, violationMessage, elapsedTime, userId, createdDate, modifiedDate, status, enricherSource)
-VALUES ('ec2_r7_vulnerability_critical','ec2_r7_vulnerability_critical','EC2WithRapid7CriticalVulnerability','Rapid7 found critical Vulnerabilities','Vulnerabilities have at least one exploit that has been used in at least one attack','','https://paladincloud.io/docs/aws-policy/#Rapid7-found-Critical-Vulnerabilities','ec2','aws','EC2WithRapid7CriticalVulnerability','{"params":[{"encrypt":"false","value":"true","key":"threadsafe"},{"key":"policyKey","value":"check-vulnerability-exists-for-server","encrypt":false},{"encrypt":false,"value":"critical","key":"severityToCheck"},{"encrypt":false,"value":"critical","key":"severity"},{"encrypt":false,"value":"security","key":"policyCategory"}],"environmentVariables":[],"policyId":"ec2_r7_vulnerability_critical","autofix":false,"policyRestUrl":"","targetType":"ec2","pac_ds":"rapid7","assetGroup":"rapid7","policyUUID":"ec2_r7_vulnerability_critical","policyType":"ManagePolicy"}','0 0 ? * MON *','','','ManagePolicy','arn:aws:events:us-east-1:***REMOVED***:rule/ec2_r7_vulnerability_critical','critical','security','false','false',NULL,24,1,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,24,'admin@paladincloud.io',now(),null,'DISABLED','rapid7');
+VALUES ('ec2_r7_vulnerability_critical','ec2_r7_vulnerability_critical','EC2WithRapid7CriticalVulnerability','Rapid7 found critical Vulnerabilities','Vulnerabilities have at least one exploit that has been used in at least one attack','','https://paladincloud.io/docs/aws-policy/#Rapid7-found-Critical-Vulnerabilities','ec2','aws','EC2WithRapid7CriticalVulnerability','{"params":[{"encrypt":"false","value":"true","key":"threadsafe"},{"key":"policyKey","value":"check-vulnerability-exists-for-server","encrypt":false},{"encrypt":false,"value":"critical","key":"severityMatchCriteria"},{"encrypt":false,"value":"critical","key":"severity"},{"encrypt":false,"value":"security","key":"policyCategory"}],"environmentVariables":[],"policyId":"ec2_r7_vulnerability_critical","autofix":false,"policyRestUrl":"","targetType":"ec2","pac_ds":"rapid7","assetGroup":"rapid7","policyUUID":"ec2_r7_vulnerability_critical","policyType":"ManagePolicy"}','0 0 ? * MON *','','','ManagePolicy','arn:aws:events:us-east-1:***REMOVED***:rule/ec2_r7_vulnerability_critical','critical','security','false','false',NULL,24,1,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,24,'admin@paladincloud.io',now(),null,'DISABLED','rapid7');
```

Comment:
Please see the story to standardize the more info url. We can use it for CS now and then standardize others

Reply:
All should point to the same url

Reply:
done